cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/TemporalFunctionsTest.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen.refcard
+
+import java.time._
+
+import org.neo4j.cypher.QueryStatisticsTestSupport
+import org.neo4j.cypher.docgen.RefcardTest
+import org.neo4j.cypher.internal.runtime.InternalExecutionResult
+
+class TemporalFunctionsTest extends RefcardTest with QueryStatisticsTestSupport {
+  val graphDescription = List("ROOT KNOWS A")
+  val title = "Temporal Functions"
+  override val linkId = "functions/temporal"
+
+  override def assert(name: String, result: InternalExecutionResult) {
+    name match {
+      case "returns-one" =>
+        assertStats(result, nodesCreated = 0)
+        assert(result.toList.size === 1)
+    }
+  }
+
+  override def parameters(name: String): Map[String, Any] = {
+    val date = LocalDate.parse("2018-04-05")
+    val time = OffsetTime.of(12, 45, 30, 250000000, ZoneOffset.ofHours(1))
+    val localtime = LocalTime.of(12, 45, 30, 250000000)
+    val datetime = java.time.ZonedDateTime.of(date, localtime, ZoneId.of("+01:00"))
+    name match {
+      case "parameters=date" =>
+        Map("year" -> 2018, "month" -> 5, "day" -> 8)
+      case "parameters=datetime" =>
+        Map("date" -> date, "time" -> time)
+      case "parameters=dateselection" =>
+        Map("datetime" -> datetime)
+      case "" =>
+        Map()
+    }
+  }
+
+  def text = """
+###assertion=returns-one
+RETURN
+
+date("2018-04-05")
+###
+Returns a date parsed from a string.
+
+###assertion=returns-one
+RETURN
+
+localtime("12:45:30.25")
+###
+
+Returns a time with no timezone.
+
+###assertion=returns-one
+RETURN
+
+time("12:45:30.25+01:00")
+###
+
+Returns a time in a specified timezone.
+
+###assertion=returns-one
+RETURN
+
+localdatetime("2018-04-05T12:34:00")
+###
+
+Returns a datetime with no timezone.
+
+###assertion=returns-one
+RETURN
+
+datetime("2018-04-05T12:34:00[Europe/Berlin]")
+###
+
+Returns a datetime in the specified timezone.
```

Comment:
"timezone" -> "time zone" (when we are speaking about time zones in general, as opposed to the timezone component)